DDOT_module/child_classes/correction_data_generator.py

```diff
@@ -0,0 +1,397 @@
+"""
+Created on 29.04.2021
+
+@author: Filip Maciejewski
+@contact: filip.b.maciejewski@gmail.com
+"""
+
+import copy
+import numpy as np
+from typing import Optional, Dict
+from tqdm import tqdm
+from QREM import ancillary_functions as anf
+from ..child_classes.ddot_marginal_analyzer_vanilla import DDOTMarginalsAnalyzer
+from DDOT_module.child_classes.global_noise_matrix_creator import GlobalNoiseMatrixCreator
+
+
+class CorrectionDataGenerator(DDOTMarginalsAnalyzer):
+    """
+        1
+    """
+
+    def __init__(self,
+                 results_dictionary_ddot: dict,
+                 reverse_counts: bool,
+                 number_of_qubits: int,
+                 marginals_dictionary: dict,
+                 clusters_list: list,
```

Comment:
Nitpick: One can use
`from typing import List`
and then write
`clusters_list: List[clusters]`
or other type.